File: src/main/java/info/devels/jtte/blocks/JTTEBlocks.java
package info.devels.jtte.blocks;

public class JTTEBlocks {
    public static BlockBeacon blockBeacon;
    public static BlockTerminal blockTerminal;

    public static void preInit() {
        blockBeacon = new BlockBeacon();
        blockTerminal = new BlockTerminal();

        blockBeacon.preInit();
        blockTerminal.preInit();
    }

    public static void initialize() {

    }

    public static void postInit() {
        blockBeacon.postInit();
        blockTerminal.postInit();
    }
}

File: src/main/kotlin/info/devels/jtte/blocks/JTTEBlocks.kt
package info.devels.jtte.blocks

import cpw.mods.fml.common.registry.GameRegistry


val blockBeacon: BlockBeacon = BlockBeacon()
val blockTerminal: BlockTerminal = BlockTerminal()
val blockCursedDirt: BlockCursedDirt = BlockCursedDirt()

fun blocksPreInit() {
    blockBeacon.preInit()
    blockTerminal.preInit()
    GameRegistry.registerBlock(blockCursedDirt, "CursedDirt")
}

fun blocksInitialize() {

}

fun blocksPostInit() {
    blockBeacon.postInit()
    blockTerminal.postInit()
}
